python/lsst/ap/verify/ap_verify.py

```diff
@@ -93,7 +92,7 @@ def __init__(self):
             self,
             description='Executes the LSST DM AP pipeline and analyzes its performance using metrics.',
             epilog='',
-            parents=[_InputOutputParser(), _ProcessingParser(), ApPipeParser(), MetricsParser()],
+            parents=[_InputOutputParser(), _ProcessingParser(), ApPipeParser(), ],
```

Comment:
Extra comma and space at the end?

Reply:
Extra comma is intentional; will fix the space. I'm surprised `flake8` didn't complain...

Reply:
Ah, `flake8` complains if you *don't* include the space. Mind if I leave it as-is? I think the comma's a useful reminder that the list is subject to change.